test/integration/files.js

```diff
@@ -12,6 +12,8 @@ const COZY_STACK_URL = process.env && process.env.COZY_STACK_URL || ''
 const COZY_STACK_VERSION = process.env && process.env.COZY_STACK_VERSION
 const COZY_STACK_TOKEN = process.env && process.env.COZY_STACK_TOKEN
 
+global.navigator = undefined
```

Comment:
I don't know why but test failed if I remove this line ;) https://travis-ci.org/cozy/cozy-client-js/builds/222804829#L640